run_imi.sh

```diff
@@ -187,17 +201,16 @@ if ! "$KalmanMode"; then
     print_stats
 fi 
 
-# copy output log to run directory for storage
+# Move output log to run directory for storage
 if [[ -f ${InversionPath}/imi_output.log ]]; then
-    cp "${InversionPath}/imi_output.log" "${RunDirs}/imi_output.log"
+    mv -v "${InversionPath}/imi_output.log" "${RunDirs}/imi_output.log"
```

Comment:
I switched it to move because I an old imi_output.log file from a previous inversion was being copied into newer IMI runs even though they crashed. I figured we could either (1) remove the imi_output.log file when submitting a new IMI run or (2) simply move the imi_output.log file after a run is complete. If you'd prefer option 1 (or neither) let me know.

Reply:
Can we do option 1? Thanks!

Reply:
Now fixed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/67f8f8a651ba6037f3cb882049755eb82ecb32c3.